docker-compose.yml

```diff
@@ -8,28 +11,41 @@ services:
     depends_on:
       - ccdc-csd-searchservice
       - csd-request-entry
-    image: ccdcrepository.azurecr.io/webcsd:0.1.6
+    image: ccdcrepository.azurecr.io/webcsd:latest
```

Comment:
The tags need to be put back to 0.1.6, but we can handle pulling in latest in the next Version which we will do shortly

Reply:
Put it back for now.